<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="BaseWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 245, 230);</string>
   </property>
   <widget class="QLabel" name="manageOption">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>351</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>option mid.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="studyOption">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>351</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>option mid.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="workRaum">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>401</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>WorkRaum</string>
    </property>
   </widget>
   <widget class="QLabel" name="managementName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>301</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 250, 255);
color: rgb(41, 171, 135);
font: 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Мой рабочий стол&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;- Заметки&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;- Записная Книжка&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="studyName">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>221</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 250, 255);
color: rgb(194, 112, 226);
font: 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Моя учёба&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;- Построение&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;графиков&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
